### ISSUE TITLE
Can't find dialog.tlk in GOG.com edition

### ISSUE THREAD

**Issue by arp242** [WeiDUorg/weidu#195]
I get this error on both BG1 and 2:

	$ cd ~/GOG\ Games/Baldurs\ Gate\ II\ Enhanced\ Edition/game

	$ ~/WeiDU-Linux/weidu --list-biffs
	[/home/martin/WeiDU-Linux/weidu] WeiDU version 24700
	[./chitin.key] 82 BIFFs, 37341 resources

	ERROR: Unable to find DIALOG.TLK in:
			./^dialog/.tlk$

	Please run this program in your Infinity Engine game directory.

	FATAL ERROR: Failure("Unable to find DIALOG.TLK")

These files do exist, but in a different location:

	$ find . -name '*.tlk'
	./lang/de_DE/dialogf.tlk
	./lang/de_DE/dialog.tlk
	./lang/fr_FR/dialogf.tlk
	./lang/fr_FR/dialog.tlk
	./lang/en_US/dialog.tlk
	./lang/ru_RU/dialogf.tlk
	./lang/ru_RU/dialog.tlk
	./lang/pl_PL/dialogf.tlk
	./lang/pl_PL/dialog.tlk
	./lang/ko_KR/dialog.tlk
	./lang/it_IT/dialogf.tlk
	./lang/it_IT/dialog.tlk
	./lang/es_ES/dialogf.tlk
	./lang/es_ES/dialog.tlk
	./lang/zh_CN/dialog.tlk

I tried using `--tlkin`, but that doesn't seem to work(?)

	$ ~/WeiDU-Linux/weidu --tlkin lang/en_US/dialog.tlk --list-biffs
	[/home/martin/WeiDU-Linux/weidu] WeiDU version 24700
	[./chitin.key] 191 BIFFs, 62570 resources

	ERROR: Unable to find DIALOG.TLK in:
			lang/en_US/dialog.tlk

	Please run this program in your Infinity Engine game directory.

	FATAL ERROR: Failure("Unable to find DIALOG.TLK")

So let's link it:

	$ ln -s lang/en_US/dialog.tlk .

And now it works:

	$ ~/WeiDU-Linux/weidu --list-biffs
	[/home/martin/WeiDU-Linux/weidu] WeiDU version 24700
	[./chitin.key] 191 BIFFs, 62570 resources
	[DATA/DEFAULT.BIF]        2850260 bytes,   1 locations
	[DATA/ITEMS.BIF]           801984 bytes,   1 locations
	[DATA/OBJANIM.BIF]       28634788 bytes,   1 locations
	[..]

**Comment by FredrikLindgren** [comment]
Yeah, I should probably add something about --use-lang to the error message.

**Comment by dark0dave** [comment]
Maybe this should not be case senstive for linux? I don't know I too had to make a link. This is not great behavior.

**Comment by dark0dave** [comment]
I guess, because I have to uppercase litterally everything in linux then lowercase it again, just to get weidu to work. This is not a great workflow.

**Comment by dark0dave** [comment]
If I remove all the links and attempt to use weidu.conf, that fails too:

```
$ cat weidu.conf
lang_dir = en_US
```
